examples/api-demo-1-list-projects.py

```diff
@@ -2,6 +2,7 @@
 
 from snyk import SnykClient
 from utils import get_token
+from utils import get_default_token_path
```

Comment:
You can merge these two: 

```
from utils import get_token, get_default_token_path
```

See the comment below though about replacing the need for the two functions with a higher-level one.

Reply:
thanks - will do

Reply:
Fixed